### PR TITLE
Add Changhao as DevTools blog post author

### PR DESCRIPTION
**Pull request by chybie**
Author data missing at https://developer.chrome.com/blog/css-grid-tooling/

### CODE REVIEW

File: site/_data/authorsData.json [hunk 1]
```diff
@@ -666,7 +666,7 @@
     "country": "DE",
     "twitter": "maudnals",
     "github": "maudnals",
-    "image": "image/O2RNUyVSLubjvENAT3e7JSdqSOx1/bpoFTthopZevDRDPPo3r.jpg"
+    "image": "image/80mq7dk16vVEg8BBhsVe42n6zn82/fJP5RRP6pcCnfctkaQhu.jpg"
```
Comment (samthor) [inline site/_data/authorsData.json:669]: is Maud's photo changing too?

**Review by samthor — APPROVED** [review]

File: site/_data/authorsData.json [hunk 2]
```diff
@@ -815,5 +815,22 @@
...
+      "en": "Google engineer working on Chrome DevTools."
+    },
+    "org": {
+      "name": "Google",
+      "unit": "Chrome"
+    },
+    "twitter": "han_changhao",
+    "github": "hanselfmu",
+    "country": "DE",
+    "image": "dPDCek3EhZgLQPGtEG3y0fTn4v82/oEV9Pd48COn9JnMrUYzZ.jpg"
```
Comment (PaulKinlan) [inline site/_data/authorsData.json:834]: The other  images have an "image/" in the path... which is correct?

File: site/_data/authorsData.json (hunk 1 above)
Comment (chybie) [reply site/_data/authorsData.json]: Nope. Revert.

File: site/_data/authorsData.json (hunk 2 above)
Comment (chybie) [reply site/_data/authorsData.json]: Good spot. Thanks.

**Pull request merged**